src/app.js

```diff
@@ -127,18 +129,37 @@ gpii.app.keyIn = function (token) {
   * @param token {String} The token to key out with.
   */
 gpii.app.keyOut = function (token) {
```

Comment:
Need to update the docs of this function so that it is clear that it initiates the keyOut action immediately and returns a promise for its completion.